sql/2022/markup/custom_elements_slot_distribution.sql

```diff
@@ -0,0 +1,32 @@
+WITH totals AS (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(0) AS total
+  FROM
+    `httparchive.summary_pages.2022_06_01_*`
+  GROUP BY
+    _TABLE_SUFFIX
+)
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  CAST(num_slots AS INT64) AS num_slots,
+  COUNT(0) AS freq,
+  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct,
+  COUNT(DISTINCT url) AS pages,
+  ANY_VALUE(total) AS total_pages,
+  COUNT(DISTINCT url) / ANY_VALUE(total) AS pct_pages
+FROM
+  `httparchive.pages.2022_06_01_*`,
+  UNNEST(JSON_VALUE_ARRAY(JSON_VALUE(payload, '$._wpt_bodies'), '$.web_components.raw.hyphenatedElements.slots')) AS num_slots
```

Comment:
`SUM(COUNT(0)) / OVER (PARTITON...` causes [problems when mixed with `UNNEST`](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide#a-warning-on-totals). You sure your total is what you think it is here? Note: note I've not checked.

Reply:
Yeah this is intentionally a count of frequencies, not pages, so it LGTM.